sections/introduction.tex

```diff
@@ -4,57 +4,46 @@ \section{Introduction}\label{sec:introduction}
 When referring to a dataset, we talk about a dataset including metadata information, which are non-essential data features.
 One example of this could be the publication date of a text document. 
 In most contexts, it is not needed to use and understand the document; however, it provides more context to the main content of the document: the text.
+
+% Intro to Nordjyske data and applications
 However, there is usually a lot of data, that is not used, either because it is not useful or we do not know how to use it properly.
+We have a dataset from a danish media group, called Nordjyske, with three years of article data.
+This dataset can be applied to a variety of different applications, such as topic modelling\cite{blei2003latent}\cite{MetaLDA2017}, text analysis\cite{baly2020written}, search engine optimization\cite{amjad2015topic}, and many more.
 
-% LDA / Topic Modeling Intro
-Within the field of topic modeling, the goal is to find underlying topics within a certain dataset.
-These topics are constructed based on the text documents in the dataset and words within these documents. 
-\Gls{lda} is a well known topic model which generates topics and topic distributions for documents based on the words in documents~\cite{blei2003latent}.
+Within the field of topic modeling, the \Gls{lda} is a well known topic model which generates topics and topic distributions for documents based on the words in documents~\cite{blei2003latent}, without the use of metadata.
 Other extensions of \gls{lda} have also modeled various other information to generate better topics and/or topics with different focuses and potential uses.
-
-% Topic Modeling dataset
-In \gls{lda}, the data used is a corpus of documents, with each document containing a sequence of words.
-However, the dataset that will form this corpus also often includes various other data than just the text within the documents, such as authors, time of publishing, tags, etc.
-% Metadata definition
-We define this other data as metadata, that is data related to the documents in the corpus, but not directly a part of the text.
-In most models, metadata is not used.
-However, there is some previous work that attempts to make full use of the dataset available by incorporating various metadata into topic models, such as the Author-Topic model by \citet{author_topic_2012} and the MetaLDA model by \citet{MetaLDA2017}.
+In most models, including extensions, metadata is not used.
```

Comment:
```suggestion
In most models metadata is not used.
```
er ikke sikker om jeg er enig med denne, da i vores brede defination af metadata er de fleste exstensions omkring en bestemt form for metadata (eg. author-topic, dynamic lda)